lib/std/build.zig

```diff
@@ -906,10 +906,11 @@ pub const Builder = struct {
         }
         const cwd = fs.cwd();
         const prev_status = try fs.Dir.updateFile(cwd, source_path, cwd, dest_path, .{});
-        if (self.verbose) switch (prev_status) {
-            .stale => warn("# installed\n", .{}),
-            .fresh => warn("# up-to-date\n", .{}),
-        };
+        if (self.verbose)
+            switch (prev_status) {
+                .stale => warn("# installed\n", .{}),
+                .fresh => warn("# up-to-date\n", .{}),
+            };
```

Comment:
Hmm I definitely prefer the previous formatting here.

Reply:
Fair enough. Just to twist your arm a *little* bit, there is an inconstancy between nested `if`s and other nested items (ie. `if`s will follow the style above, but `switch`s will follow the status quo style).
I'll revert the fix, and #6114 should be rejected.

Reply:
Hmm I see your point (and I'm glad you're pushing back btw). Can we stick with status quo for the purposes of this PR and then discuss it further in that issue?

Reply:
We certainly can